Translated/Dialog/Rasta/ssn_014355.csv

```diff
@@ -1,10 +1,10 @@
 name_7719#0,"""Ratel"""
-ssn_014355_0010#0,"""Hmm, just what could be up with this place?<br>Huh? You've been looking for me?"""
-ssn_014355_0020#0,"""So Rasta sent you, did she? I'm sorry if I made her worry. I was investigating these Ore veins, since I was outside the City already."""
-ssn_014355_0030#0,"""As you can see, I'm in no trouble here.<br>But my partner, though..."""
+ssn_014355_0010#0,"""Oh, what are you doing all the way out here?<br>Huh? You're looking for me?"""
```

Comment:
For this line, "you've been" seems more accurate to the situation. You aren't currently looking for someone if you've just found them.